### DISCUSSION TITLE
running plover from source without installing PyQt

### DISCUSSION THREAD

**Discussion by Josiah-tan** [opensteno/plover#1572] [Q&A and Support]
on something like a Raspberry Pi, it is very hard to install PyQt.
So I am wondering if it is possible to run plover from source without installing this because the gui is not necessarily needed (like in stenogochi)

**Comment by benoit-pierre** [comment]
No, see below. ~Yes, the Qt GUI is provided through the optional `gui_qt` extra, so just pip install without it.~

**Reply by Josiah-tan** [reply]
https://github.com/openstenoproject/plover/blob/3066a9a47269861ac1d66f8f05cdb26f7251b98d/tox.ini#L27
so I just comment out this line and run `tox test`?

**Comment by benoit-pierre** [comment]
OK, let me rephrase: running from the git checkout is for Plover development, so you need everything (including PyQt5).

Otherwise, including for plugins development, you can install Plover from a wheel, which by default does not pull in the GUI dependencies.

As for PyQt5 being hard to install, I'm surprised the piwheels guys still have not fixed their PyQt5 builds...

Is the official PyQt5 distribution package still missing the required metadata? If that's still the case, the script [mentioned here](https://github.com/openstenoproject/plover/discussions/1430#discussioncomment-1566444) might help.
